### PR TITLE
Better error for missing vectorizer

### PR DESCRIPTION
**Pull request by dirkkul**
### What's being changed:

Closes https://github.com/weaviate/weaviate/issues/4976, https://github.com/weaviate/weaviate/issues/4523

Finally improve the error message for this

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: usecases/modules/modules.go [hunk 1]
```diff
@@ -723,7 +723,7 @@ func (p *Provider) VectorFromSearchParam(ctx context.Context,
 		}
 	}
 
-	panic("VectorFromSearchParam was called without any known params present")
+	return nil, "", fmt.Errorf("could not vectorize input for collection %v with search-type %v and parameters %v. Make sure a vectorizer module is configured for this class", className, param, params)
```
Comment (databyjp) [inline usecases/modules/modules.go:726]: Should `class` be `collection`?

Comment (databyjp) [inline usecases/modules/modules.go:726]: Makes sense. I propose `class` -> `collection`

Minor suggestion: 

Would it be worth surfacing the targetVector? e.g.: 
`fmt.Errorf("could not vectorize input for collection %v (target vector %v) with search-type %v and parameters %v. Make sure a vectorizer module is configured for this class", className, targetVector, param, params)`

File: usecases/modules/modules.go [hunk 2]
```diff
@@ -752,7 +752,7 @@ func (p *Provider) CrossClassVectorFromSearchParam(ctx context.Context,
 		}
 	}
 
-	panic("CrossClassVectorFromSearchParam was called without any known params present")
+	return nil, "", fmt.Errorf("could not vectorize input for Explor with search-type %v and parameters %v. Make sure a vectorizer module is configured", param, params)
```
Comment (databyjp) [inline usecases/modules/modules.go:755]: `Explor` -> `Explore`

LGTM.

File: usecases/modules/modules.go (hunk 1 above)
Comment (dirkkul) [reply usecases/modules/modules.go]: changed :)

File: .github/workflows/pull_requests.yaml [hunk 3]
```diff
@@ -355,6 +355,7 @@ jobs:
       CLUSTER_HOSTNAME: node1
       RAFT_BOOTSTRAP_EXPECT: 1
       RAFT_JOIN: node1
+      GRPC_PORT: 50052
```
Comment (antas-marcin) [inline .github/workflows/pull_requests.yaml:358]: we need to change the `gRPC` port here, yes?

**Review by antas-marcin — APPROVED** [review]

**Pull request merged**